astro.config.mjs

```diff
@@ -2,6 +2,8 @@ import { defineConfig } from "astro/config";
 import starlight from "@astrojs/starlight";
 import starlightLinksValidator from "starlight-links-validator";
 
+import mdx from "@astrojs/mdx";
```

Comment:
Had issues rendering the blog posts written in MDX until this was installed.
See https://docs.astro.build/en/guides/upgrade-to/v5/#astrojsmdx